src/hb-aat.h

```diff
@@ -0,0 +1,540 @@
+  HB_AAT_LAYOUT_SELECTOR_SCIENTIFIC_INFERIORS		= 4,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_FRACTIONS
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_NO_FRACTIONS			= 0,
+  HB_AAT_LAYOUT_SELECTOR_VERTICAL_FRACTIONS		= 1,
+  HB_AAT_LAYOUT_SELECTOR_DIAGONAL_FRACTIONS		= 2,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_OVERLAPPING_CHARACTERS_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_PREVENT_OVERLAP_ON		= 0,
+  HB_AAT_LAYOUT_SELECTOR_PREVENT_OVERLAP_OFF		= 1,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_TYPOGRAPHIC_EXTRAS
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_HYPHENS_TO_EM_DASH_ON   = 0,
+  HB_AAT_LAYOUT_SELECTOR_HYPHENS_TO_EM_DASH_OFF  = 1,
+  HB_AAT_LAYOUT_SELECTOR_HYPHEN_TO_EN_DASH_ON    = 2,
+  HB_AAT_LAYOUT_SELECTOR_HYPHEN_TO_EN_DASH_OFF   = 3,
+  HB_AAT_LAYOUT_SELECTOR_SLASHED_ZERO_ON         = 4,
+  HB_AAT_LAYOUT_SELECTOR_SLASHED_ZERO_OFF        = 5,
+  HB_AAT_LAYOUT_SELECTOR_FORM_INTERROBANG_ON     = 6,
+  HB_AAT_LAYOUT_SELECTOR_FORM_INTERROBANG_OFF    = 7,
+  HB_AAT_LAYOUT_SELECTOR_SMART_QUOTES_ON         = 8,
+  HB_AAT_LAYOUT_SELECTOR_SMART_QUOTES_OFF        = 9,
+  HB_AAT_LAYOUT_SELECTOR_PERIODS_TO_ELLIPSIS_ON  = 10,
+  HB_AAT_LAYOUT_SELECTOR_PERIODS_TO_ELLIPSIS_OFF = 11,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_MATHEMATICAL_EXTRAS
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_HYPHEN_TO_MINUS_ON       = 0,
+  HB_AAT_LAYOUT_SELECTOR_HYPHEN_TO_MINUS_OFF      = 1,
+  HB_AAT_LAYOUT_SELECTOR_ASTERISK_TO_MULTIPLY_ON  = 2,
+  HB_AAT_LAYOUT_SELECTOR_ASTERISK_TO_MULTIPLY_OFF = 3,
+  HB_AAT_LAYOUT_SELECTOR_SLASH_TO_DIVIDE_ON       = 4,
+  HB_AAT_LAYOUT_SELECTOR_SLASH_TO_DIVIDE_OFF      = 5,
+  HB_AAT_LAYOUT_SELECTOR_INEQUALITY_LIGATURES_ON  = 6,
+  HB_AAT_LAYOUT_SELECTOR_INEQUALITY_LIGATURES_OFF = 7,
+  HB_AAT_LAYOUT_SELECTOR_EXPONENTS_ON             = 8,
+  HB_AAT_LAYOUT_SELECTOR_EXPONENTS_OFF            = 9,
+  HB_AAT_LAYOUT_SELECTOR_MATHEMATICAL_GREEK_ON    = 10,
+  HB_AAT_LAYOUT_SELECTOR_MATHEMATICAL_GREEK_OFF   = 11,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_ORNAMENT_SETS_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_NO_ORNAMENTS          = 0,
+  HB_AAT_LAYOUT_SELECTOR_DINGBATS              = 1,
+  HB_AAT_LAYOUT_SELECTOR_PI_CHARACTERS         = 2,
+  HB_AAT_LAYOUT_SELECTOR_FLEURONS              = 3,
+  HB_AAT_LAYOUT_SELECTOR_DECORATIVE_BORDERS    = 4,
+  HB_AAT_LAYOUT_SELECTOR_INTERNATIONAL_SYMBOLS = 5,
+  HB_AAT_LAYOUT_SELECTOR_MATH_SYMBOLS          = 6,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_CHARACTER_ALTERNATIVES
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_NO_ALTERNATES = 0,
+/*
+ * Selectors for #HB_AAT_LAYOUT_FEATURE_TYPE_DESIGN_COMPLEXITY_TYPE
+ *
+ * Since: REPLACEME
+ */
+  HB_AAT_LAYOUT_SELECTOR_DESIGN_LEVEL1 = 0,
+  HB_AAT_LAYOUT_SELECTOR_DESIGN_LEVEL2 = 1,
+  HB_AAT_LAYOUT_SELECTOR_DESIGN_LEVEL3 = 2,
+  HB_AAT_LAYOUT_SELECTOR_DESIGN_LEVEL4 = 3,
+  HB_AAT_LAYOUT_SELECTOR_DESIGN_LEVEL5 = 4,
+/*
```

Comment:
Remove the empty line from beginning of comment, add real empty line before.

Reply:
Was just a draft, fixed